src/main/java/racingcar/RacingCar.java

```diff
@@ -0,0 +1,29 @@
+package racingcar;
+
+public class RacingCar {
+
+    public void nameCheck(String carName) {
+        String carNameArr[] = carName.split(",");
```

Comment:
```suggestion
        String[] carNames = carName.split(",");
```
위와 같이 변경해보면 어떨까요? 
[좋은 코드를 위한 자바 변수명 네이밍](https://woowacourse.github.io/javable/post/2020-04-24-variable_naming/) 

또한 [구글 코드 컨벤션](https://google.github.io/styleguide/javaguide.html#s4.8.3-arrays)에 따르면 배열의 선언은 아래와 같이 하라고 가이드하고 있어요 😉
```suggestion
        String[] carNameArr = carName.split(",");
```